extension.js

```diff
@@ -154,17 +168,14 @@ class HuaweiWmiIndicator extends PanelMenu.Button { // TODO: move to system batt
 	}
 
 	_set_bpm(low, high) {
-		let file_sys = Gio.File.new_for_path("/sys/devices/platform/huawei-wmi/charge_control_thresholds");
-		let file_def = Gio.File.new_for_path("/etc/default/huawei-wmi/charge_control_thresholds");
-
 		if (low || high)
 			try {
-				file_sys.replace_contents(`${low} ${high}`, null, false, 0, null);
-				file_def.replace_contents(`${low} ${high}`, null, false, 0, null);
+				this._file_sys.replace_contents(`${low} ${high}`, null, false, 0, null);
```

Comment:
Would this still work after the module reinserted without reopening the file?

Reply:
I don't know what you mean about reloading the module but the files stays open as instance variables on line 43 and 44. It is done this way since there is now multiple methods where the files are going to be used.

One issue that I see now is that the If on 160/171 is not wrapped in a block..

How would I go about editing these requested changes. Do I just commit them to my fork?

Reply:
I mean `rmmod`'ing then `modprobe`'ing the [kernel module](https://github.com/aymanbagabas/Huawei-WMI), e.g. during its development. The file descriptor will then become stale and unusable, that's the exact reason why I reopen them each time in the first place.

The `if` on L171 does not need its content wrapped, as it's a single statement (a `try..catch` block). Per my code style, the braces are redundant in this case. Please remove them, too.

And yes, you should just commit and push your changes to the same branch — GitHub will handle the rest.

Reply:
If that is the case then I'm going to rework the code to use the previous method of loading files. I did not even realise that this could be an issue as I'm still a novice. Thank you for pointing this out :)

Reply:
Okay, that's kind of correct now. I'll probably rework variable naming later; good for now.